packages/ariakit/src/menu/menu-item-checkbox.ts

```diff
@@ -1,7 +1,7 @@
-import { useCallback } from "react";
+import { MouseEvent as ReactMouseEvent, useCallback } from "react";
```

Comment:
Let's use the original name. But maybe we can reuse the MenuItemOptions type (suggestion below).
```suggestion
import { useCallback } from "react";
```

Reply:
:+1: yeah I agree reusing the `MenuItemOptions` type is cleaner since we're passing it through anyway.

Reply:
:+1: yeah I agree reusing the `MenuItemOptions` type is cleaner since we're passing it through anyway.